Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ValidatorData.xml

```diff
@@ -5622,4 +5622,48 @@
         <WhichUnit Value="Source"/>
         <Value value="AP_PhoenixTaldarim"/>
     </CValidatorUnitType>
+    <CValidatorPlayerRequirement id="AP_HaveTyrannozorTyrantsProtection">
+        <Find value="1"/>
+        <Value value="AP_HaveTyrannozorTyrantsProtection"/>
+    </CValidatorPlayerRequirement>
+    <CValidatorLocationEnumArea id="AP_TyrannozorBarrageOfSpikesEnemiesNearby">
+        <AreaArray Compare="GE" Count="1" Radius="2"/>
+        <SearchFilters value="Visible;Self,Player,Ally,Neutral,Air,Structure,RawResource,HarvestableResource,Missile,Stasis,Dead,Hidden,Invulnerable"/>
+    </CValidatorLocationEnumArea>
+    <CValidatorPlayerRequirement id="AP_HaveTyrannozorMerge">
+        <Find value="1"/>
+        <Value value="AP_HaveTyrannozorMerge"/>
+    </CValidatorPlayerRequirement>

```

Comment:
This validator seems unused?

Reply:
Yeah, no idea why I even wrote this, probably some throw it at the wall idea for fixing merges that I forgot to cleanup